src/arch/xtensa/xtos/stub.c

```diff
@@ -0,0 +1,99 @@
+// SPDX-License-Identifier: BSD-3-Clause
+#include <stddef.h>
+#include <sys/types.h>
+#include <sys/stat.h>
+#include <sof/debug/panic.h>
+#include <ipc/trace.h>
+
+struct _reent;
+
```

Comment:
@lkoenig could we have a comment at the top on the usage, i.e. that this is needed for linkage only for certain modules. This will make it clear to any developers who come across the stub.

Reply:
I added something. Let me know if it fits the bill